### ISSUE TITLE
alt=""

### ISSUE THREAD

**Issue by sehyunchung** [sxhyxnchxng/til#54]
https://developers.google.com/web/fundamentals/accessibility/semantics-builtin/text-alternatives-for-images

<q>To summarize, all images should have an alt attribute, but they need not all have text. Important images should have descriptive alt text that succinctly describes what the image is, while decorative images should have empty alt attributes — that is, alt="".</q>
